halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/persistentStorage/PersistentStorageValidator.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.config.validate.v1.persistentStorage;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.*;
+import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
+import com.netflix.spinnaker.halyard.core.problem.v1.Problem.Severity;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PersistentStorageValidator extends Validator<PersistentStorage> {
+  @Override
+  public void validate(ConfigProblemSetBuilder ps, PersistentStorage n) {
+    NodeIterator nodeIterator = n.getChildren();
+
+    int numEnabled = 0;
+    Node child = nodeIterator.getNext();
+    while (child != null) {
+      PersistentStore persistentStore = (PersistentStore) child;
+      if (persistentStore.enabled) {
+        numEnabled++;
+      }
+
+      child = nodeIterator.getNext();
+    }
+
+    if (numEnabled == 0) {
+      ps.addProblem(Severity.FATAL, "No persistent store is enabled.")
+          .setRemediation("Enable a persistent store.");
```

Comment:
I don't think these remediations add much beyond what the problem message is telling the user.